cumulusci/salesforce_api/package_install.py

```diff
@@ -62,6 +64,7 @@ class PackageInstallOptions(CCIModel):
     security_type: SecurityType = SecurityType.FULL
     apex_compile_type: Optional[ApexCompileType] = None
     upgrade_type: Optional[UpgradeType] = None
+    skip_handlers: SkipHandlers = SkipHandlers.FEATURE_ENFORCEMENT
```

Comment:
±: We don't want to silently change behavior, so it shouldn't be a default. Instead, it should be an optional `Option` set to `None`.

Reply:
Updated to `None`